packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -490,7 +486,7 @@ export class FluidDataStoreRuntime
 	public bindChannel(channel: IChannel): void {
 		assert(
 			this.notBoundedChannelContextSet.has(channel.id),
-			0x17b /* "Channel to be binded should be in not bounded set" */,
+			0x17b /* "Channel to be bound should be in not bounded set" */,
```

Comment:
not sure how much we care, but maybe worth also updating the associated assert tooling files. (i think people always search these by assert code anyway and if we don't already have tests for this assert which fail i doubt we'll add them in the future)

Reply:
at next release it will auto update, and this is an old assert, so i don't imagine anyone would look for it via anything but the code.